src/game/BattleGround/BattleGroundWS.h

```diff
@@ -101,9 +111,92 @@ enum BG_WS_Events
     WS_EVENT_SPIRITGUIDES_SPAWN   = 2
 };
 
+static const uint32 wsFlagIds[PVP_TEAM_COUNT] = { 
```

Comment:
It looks like none of the statics in this new block are used in the header, so please move all of this to the source file (for example to the start of the source file, in an anonymous namespace). I know this project isn't always the best example, but it's generally good practice to keep header files clean and minimal, so everything that's not part of "the interface" or "the API" and that can reasonably be kept out of headers, should probably not be in the header.